### PR TITLE
Add example on theme definition and change themes

### PR DESCRIPTION
**Pull request by xevifar**

### CODE REVIEW

File: Examples/Style/Android/KeyboardStyle/app/.gitignore
Comment (iagotorrado-fleksy) [inline Examples/Style/Android/KeyboardStyle/app/.gitignore:1]: Please add this to the gitignore and do a refresh: https://github.com/github/gitignore/blob/main/Android.gitignore
Refresh gitignore: https://sigalambigha.home.blog/2020/03/11/how-to-refresh-gitignore/

Comment (xevifar) [reply Examples/Style/Android/KeyboardStyle/app/.gitignore:1]: Nice! Done

File: Examples/Style/Android/KeyboardStyle/app/src/main/java/co/thingthing/sample/sdksample/SampleKeyboardService.kt [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            boolTheme = true
+        }
+    }
+
+    // Add here the leadingTopView and the trailingTopView.
+    // override val leadingTopBarView: View?
+    //    get() = super.leadingTopBarView
+    //
+    // override val trailingTopBarView: View?
+    //    get() = super.trailingTopBarView
```
Comment (iagotorrado-fleksy) [inline Examples/Style/Android/KeyboardStyle/app/src/main/java/co/thingthing/sample/sdksample/SampleKeyboardService.kt:137]: This commented block can be removed.

File: Examples/Style/Android/KeyboardStyle/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.aar'])
+    implementation 'androidx.appcompat:appcompat:1.6.1'
+    implementation 'androidx.core:core-ktx:1.12.0'
+    implementation "androidx.constraintlayout:constraintlayout:2.1.4"
+    testImplementation 'junit:junit:4.13.2'
+    implementation 'com.android.support:multidex:1.0.3'
+
+	// Keyboard Dependencies
+	implementation "co.thingthing.fleksycore:fleksycore-release:4.7.2"
```
Comment (iagotorrado-fleksy) [inline Examples/Style/Android/KeyboardStyle/app/build.gradle:65]: Version 4.7.4 is available if you want to upgrade

**Review by iagotorrado-fleksy — APPROVED** [review]

File: Examples/Style/Android/KeyboardStyle/app/src/main/java/co/thingthing/sample/sdksample/SampleKeyboardService.kt (hunk 1 above)
Comment (xevifar) [reply Examples/Style/Android/KeyboardStyle/app/src/main/java/co/thingthing/sample/sdksample/SampleKeyboardService.kt:137]: I was intended to show that a developer can add a "leadingView" or "trailingView". Just to be consistent on the iOS example that we show the same too.

File: Examples/Style/Android/KeyboardStyle/app/build.gradle (hunk 2 above)
Comment (xevifar) [reply Examples/Style/Android/KeyboardStyle/app/build.gradle:65]: Completely, Updated, thanks!

**Pull request merged**